docs/report.md

```diff
@@ -201,7 +202,7 @@
 | munmap                  | 🟢        | aarch64,loongarch64,ppc64le,riscv64,s390x,x86_64 |
 | name_to_handle_at       | 🟡        | aarch64,loongarch64,ppc64le,riscv64,s390x,x86_64 |
 | nanosleep               | 🟢        | aarch64,loongarch64,ppc64le,riscv64,s390x,x86_64 |
-| newfstatat              | 🟢        | aarch64,ppc64le,riscv64,s390x,x86_64             |
```

Comment:
they are removing the syscall, that's pretty weird :/

Reply:
Nope they moved to `fstatat` :D

Reply:
yep the glibc wrapper will probably now point to `fstatat` but what if i call `newfstatat` directly?

Reply:
that's a nice question; indeed some of our tests are failing now :/